### DISCUSSION TITLE
New Update Waveshare 2.13 v3 Blank

### DISCUSSION THREAD

**Discussion by FusedStamen** [jayofelony/pwnagotchi#253] [Q&A]
I updated two of my devices and I can see the device is working via webui, but the screen doesn't work at all. Not sure what I'm missing - I restored a backup of a config that had previously worked

**Comment by jayofelony** [comment]
What if you only add the ui.display stuff in the config.toml?

**Reply by FusedStamen** [reply]
I'll have to give this a try, I posted a few things in the Discord as well but I'm at work without access to it really for the day. Thank you! I'll follow up with how it works.

**Reply by FusedStamen** [reply]
That didn't seem to work, I'm going to try a brand new image and not restore at all and see how it works with a fresh image on its own.

**Comment by FusedStamen** [comment]
it looks like either brain or brain references in the config broke things, restoring slowly without restoring config looks to have helped
